docs/contributing/making-changes-to-a-pull-request.md

```diff
@@ -0,0 +1,17 @@
+# Making changes to a Pull Request
+
+When a Pull Request branch is [checked out](review-a-pull-request-in-visual-studio.md) you can commit changes to it and push and pull like any other branch. If the Pull Request branch is located in a fork and was checked out from the GitHub Pull Request details pane, then a remote to that fork will be created automatically and the branch set to track the fork branch.
```

Comment:
> a Pull Request branch

I've not often heard this called a "pull request branch"; it seems more accurate to call it a topic branch.